AIX compiling from source is failing

operating system: AIX 7.3
compiler: gcc 8.3.0
applied build instructions: [Building with configure (*nix including GNU/Linux)](https://github.com/universal-ctags/ctags/blob/master/docs/autotools.rst#building-with-configure-nix-including-gnulinux)
error when linking:

```
...
gcc -std=gnu99 -Wall   -I/opt/freeware/include/libxml2 -I/opt/freeware/include     -g -O2   -o ctags  main/ctags-cmd.o libctags.a gnulib/libgnu.a -pthread -L/opt/freeware/lib64 -lxml2    -liconv
ld: 0711-317 ERROR: Undefined symbol: ._isnan
ld: 0711-345 Use the -bloadmap or -bnoquiet option to obtain more information.
collect2: error: ld returned 8 exit status
...
```

solution:

```
LIBS=-lm ./configure 
```

question: any idea how this additional LIBS environment configuration can be avoided?

--- a/configure.ac
+++ b/configure.ac
@@ -848,6 +848,8 @@ fi
 # Output files
 # ------------
 
+AC_CHECK_LIB([m], [isnan])
+
 AC_CHECK_FUNCS(mblen)
 
 AC_CONFIG_FILES([Makefile gnulib/Makefile man/GNUmakefile])

